packages/formvuelate/src/SchemaFormFactory.js

```diff
@@ -22,6 +22,18 @@ export default function SchemaFormFactory (plugins = [], components = null) {
     }
   }
 
+  const formProps = { ...SchemaForm.props }
+
+  function extendProps (newProps = {}) {
+    Object.assign(formProps, newProps || {})
+  }
+
+  plugins.forEach(plugin => {
+    if (plugin.beforeSetup) {
+      plugin.beforeSetup({ extendProps })
```

Comment:
I dont dislike `beforeSetup`, but we could also just say `plugin.extend`?

Reply:
`extend` or `extendSchemaForm` was on top of my list. `beforeSetup` may imply that this is a Vue cycle when it is not so I would like to avoid it.

Reply:
I really like `extendSchemaForm`, I'm all for verboseness for clarity (`preventmodelcleanuponschemachange` 🤣 ) 
UNLESS we expect it at some point to extend something else because then it would be a breaking change, so that's something to keep in mind